logfire/_internal/integrations/fastapi.py

```diff
@@ -294,8 +294,11 @@ async def run_endpoint_function(
                 # both `http.method` and `http.route`.
                 **{'http.route': request.scope['route'].path},
                 **stack_info,
-                _level='debug',
             )
+            # This span is level debug mostly for legacy reasons.
+            # We set the level after starting so that it's not excluded by the min_level setting,
+            # since this span is requested explicitly.
+            extra_span.set_level('debug')
```

Comment:
Is this really the desired behavior? I'm thinking if you are opting into having this span, and have min level info (rather than debug), maybe we should just make this span be info level? It feels weird to me that I can set min level to be info, and yet still get debug spans coming through as long as they are the _right_ debug spans..

Reply:
I don't know the right answer. I was just hesitant about changing the level of the span.